tests/run.php

```diff
@@ -323,6 +316,10 @@
 $nestedSet->addTestCase(new Doctrine_NestedSet_Hydration_TestCase());
 $test->addTestCase($nestedSet);
 
+// This test seems to always cause issues with other tests due to the way it loads models I believe
+// moving to last test in the suite
+$test->addTestCase(new Doctrine_Base_TestCase());
```

Comment:
This appears to be the true fix to my "getting tests to pass on Github Actions" woes, despite all the other changes.

There's something about the way this test case loads the models that causes other tests to fail (but only in some environments apparently).